target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4019-m019u0.dtsi

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
```

Comment:
do we need an extra dtsi here ? can't we reuse an existing one ?

Reply:
yes, there are 2 more board use the dtsi,i`ll make it done later